runtime/Cpp/runtime/src/ANTLRInputStream.cpp

```diff
@@ -144,7 +153,11 @@ std::string ANTLRInputStream::getText(const Interval &interval) {
     return "";
   }
 
-  return antlrcpp::utf32_to_utf8(_data.substr(start, count));
+  auto maybe_utf8 = Utf8::strictEncode(std::u32string_view(_data).substr(start, count));
+  if (!maybe_utf8.has_value()) {
+    throw IllegalArgumentException("Input stream contains invalid Unicode code points");
+  }
+  return std::move(maybe_utf8).value();
```

Comment:
Please stay with the naming convention used in the rest of the code: camelCase.

Reply:
Done. Sorry, we use a different style and sometimes I forget.